src/utils/spotify/spotify.utils.js

```diff
@@ -43,7 +41,8 @@ const getSpotifyToken = async () => {
 
 app.use(express.json());
 
-const token = await getSpotifyToken();
+refreshSpotifyToken();
+setInterval(refreshSpotifyToken, 30 * 50 * 1000);
```

Comment:
It would be worthwhile to handle this setInterval with 
`
const **tokenRefreshIntervalHandle** = setInterval(...);

// In app.tsx or other component that relies on the token refresh...

useEffect(() => {
//  ... other code
 return () => {
**clearInterval(tokenRefreshIntervalHandle);**
}
}, [])

`

Reply:
Moreover I'd extract this calculation to `REFRESH_TOKEN_INTERVAL_MS = 30 * 50 * 1000`
and would use it while setting the interval e.g.
`setInterval(refreshSpotifyToken, REFRESH_TOKEN_INTERVAL_MS);`

Alternatively, I'd even skip the calculation to save some CPU cycles and just go:
`REFRESH_TOKEN_INTERVAL_MS = 150000 // 30 (units) * 50 (units) * 1000 (ms) - specifying units of the magic numbers, so the other person can arrive at the same calculation`